nixos/modules/virtualisation/qemu-vm.nix

```diff
@@ -198,103 +212,35 @@ let
 
   regInfo = pkgs.closureInfo { rootPaths = config.virtualisation.additionalPaths; };
 
-
   # Generate a hard disk image containing a /boot partition and GRUB
   # in the MBR.  Used when the `useBootLoader' option is set.
   # Uses `runInLinuxVM` to create the image in a throwaway VM.
   # See note [Disk layout with `useBootLoader`].
```

Comment:
This is not really accurate anymore. The Disk Layout note is gone and the image might not contain grub, right?

Reply:
Right!